packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Writers/CodeWriter.cs

```diff
@@ -312,20 +320,21 @@ public void WriteProperty(PropertyDeclaration property)
                     AppendRaw(";");
                     break;
                 case AutoPropertyBody(var hasSetter, var setterModifiers, var initialization):
-                    AppendRaw("{ get; ");
+                    AppendRaw("{ get;");
                     if (hasSetter)
                     {
                         WritePropertyAccessorModifiers(setterModifiers);
-                        AppendRaw(" set; ");
+                        AppendRaw("set;");
                     }
-                    AppendRaw("}");
+                    AppendRaw(" }");
```

Comment:
well I changed the spaces here so that we could have a readable spaces in the test cases.
The generated code in test cases has not been formatted therefore the previous spaces will lead to a multiple spaces in a row.
As for the real generated code, we have roslyn to format them therefore there is no space issues.